spyder_notebook/server/notebook-template.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
Could you add in a comment above this line why this template is necessary? That's not clear from the content below.